deployment/scripts/sar.py

```diff
@@ -0,0 +1,96 @@
+import json
```

Comment:
Please add `#!/usr/bin/env python` at the top of the file and make it executable.